src/renderer/lib/cast.js

```diff
@@ -130,22 +132,51 @@ function chromecastPlayer () {
     })
   }
 
+  function serveSubtitles (cb) {
+    const subtitles = state.playing.subtitles
+    const selectedSubtitle = subtitles.tracks[subtitles.selectedIndex]
+    if (!selectedSubtitle) {
+      cb()
+    } else {
+      ret.subServer = http.createServer((req, res) => {
+        if (!selectedSubtitle) {
```

Comment:
I don't think this code will ever be reached because `selectedSubtile` has already been checked for a value in the above `else` clause.